check-ntpoffset/lib/check-ntpoffset.go

```diff
@@ -206,3 +211,65 @@ func parseNTPOffsetFromChrony(out io.Reader) (offset float64, err error) {
 	}
 	return 0.0, fmt.Errorf("failed to get ntp offset")
 }
+
+func getNTPStratum() (string, error) {
+	ntpdName, err := detectNTPDname()
+	if err != nil {
+		return "unknown", err
```

Comment:
I think it is sufficient to return an empty string `""` instead of `"unknown"` when error occurred, because it will not be used at all.

Reply:
Thank you for pointing it out.
I've replace `unknown` with  empty string.